lib/suspenders/actions/strip_comments_action.rb

```diff
@@ -0,0 +1,203 @@
+require "parser/current"
+
+module Suspenders
+  module Actions
+    class StripCommentsAction
+      class << self
+        def call(source)
+          parser = Parser::CurrentRuby.new
+
+          source
+            .yield_self { |s| strip_comments(s, parser) }
```

Comment:
Do we need to support old Ruby versions? If we don't, what about using `then` here. I love how it reads
```suggestion
            .then { |src| strip_comments(src, parser) }
```

Reply:
@MatheusRich Good call! `then` was introduced in Ruby 2.6 so we should be safe here. We are not supposed to support old Ruby versions. Thank you for that, I'll replace it with `then`.

Reply:
About the variable name, IMO `s` is perfectly good and succinct because it can be inferred from the context that we're dealing with `source`.